packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -1075,7 +1125,15 @@ export class PaginatedMessage {
 			type: Constants.MessageComponentTypes.BUTTON,
 			run: async ({ collector, response }) => {
 				collector.stop();
-				await response.edit({ components: [] });
+				if (runsOnInteraction(response)) {
+					if (response.replied || response.deferred) {
+						await response.editReply({ components: [] });
+					} else {
+						await response.reply({ components: [] });
```

Comment:
Ditto

Reply:
Same here as above. It's cleanup of the stored `Message | CommandInteraction | SelectMenuInteraction`, no Button interactions here.